rpcd/playbooks/roles/rpc_maas/templates/network_throughput.yaml.j2

```diff
@@ -1,4 +1,4 @@
-{% set max_speed = (item.0.max_speed | default(0, boolean=true) or item.1.stdout) | int * 10 ** 6 / 8 %}
+{% set max_speed = (item.0.max_speed | default(item.1.stdout | default(1000, boolean = true), boolean = true)) | int * 10 ** 6 / 8 %}
```

Comment:
Support should never need to set this value, if a value isn't automatically discovered something is probably broken and so it is better that they are notified than a default value gets popped in. It is simple enough to set the variable in the overrides file if required.